src/main/java/io/cdap/e2e/pages/locators/CdfConnectionLocators.java

```diff
@@ -94,4 +95,23 @@ public static WebElement locateConnectionAction(String connectionType, String co
 
   @FindBy(how = How.XPATH, using = "//div[@data-cy='connection-test-failure']")
   public static WebElement locateConnectionFooterError;
+
+
+  @FindBy(how = How.XPATH, using = "//img[@id='resource-center-btn']")
+  public static WebElement plusGreenButton;
+
+  public static By importuploadButton() {
```

Comment:
You have already created the import button locator below. What is this one doing ?

Reply:
One is just for finding the element and the second locator is just used for uploading the json file